server/config/config.go

```diff
@@ -803,6 +805,7 @@ const (
 	defaultSplitMergeInterval        = time.Hour
 	defaultSwitchWitnessInterval     = time.Hour
 	defaultEnableDiagnostic          = false
+	defaultOperatorTimeoutOffset     = 0 * time.Second
```

Comment:
default close?

Reply:
yes, It may be used in big cluster such region heartbeat bottleneck .